src/libraries/System.IO.FileSystem/tests/DisabledFileLockingTests/System.IO.FileSystem.DisabledFileLocking.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <!-- file locking can't be disabled on Windows -->
+    <TargetFrameworks>$(NetCoreAppCurrent)-Unix</TargetFrameworks>
```

Comment:
Can we instead have this run everywhere, and have the test changed to be:
Assert.Equal(OperatingSystem.IsWindows(), PlatformDetection.IsFileLockingEnabled);
?

Reply:
This project includes all test files that are run for `System.IO.FileSystem.Tests.csproj`. If we enable it for Windows, we are basically going to run the same tests with the same config twice on Windows. Why would we want to do that?

Reply:
Why do we want to do it on Unix? The setting controls a minority of functionality. Shouldn't we be more scoped?

Regardless,  my point was that your comment asserts this property,  which now controls whether multiple tests run,  will always be true on Windows. Seems like a good thing to validate with the test you're adding.

Reply:
> Why do we want to do it on Unix?

My main motivation is to make sure that file IO actually works as expected when the locking is disabled. It's unlikely that we introduce a bug in this code path, but it's still possible.

Reply:
Sure, but running all 6000+ tests?